Mathlib/SetTheory/Surreal/Basic.lean

```diff
@@ -202,6 +202,32 @@ theorem numeric_of_isEmpty_rightMoves (x : PGame) [IsEmpty x.RightMoves]
   Numeric.mk (fun _ => isEmptyElim) H isEmptyElim
 #align pgame.numeric_of_is_empty_right_moves SetTheory.PGame.numeric_of_isEmpty_rightMoves
 
+/-- Inserting a smaller numeric left option into a numeric results in a numeric. -/
+theorem numeric_of_insertLeft_numeric {x x' : PGame} (x_num : x.Numeric) (x'_num : x'.Numeric)
```

Comment:
Name is wrong. Should be `insertLeft_numeric_of_numeric`, but just `insertLeft_numeric` is better. 

Similarly below.